### ISSUE TITLE
Model is black on AR example

### ISSUE THREAD

**Issue by takahirox** [google/model-viewer#1016]
### Description

I tried [model-viewer AR example](https://modelviewer.dev/examples/augmented-reality.html) on my Samsung Galaxy S8+ + Chrome Canary. I see a model black (I guess no lighting?) in AR mode while the model looks fine out of AR mode. Is this a known issue? Or can it be a platform specific issue?

![image](https://user-images.githubusercontent.com/7637832/74285350-810e4900-4cda-11ea-99fd-d31e32854477.png)

Device: Samsung Galaxy S8+
OS: Android
Browser: Chrome Canary 82.0.4054.0

Console log via remote debugger
```javascript
prismatic.min.js:6 Unable to render content: No mixed-reality browser detected.
three.module.js:16000 THREE.WebGLRenderer: OES_texture_float extension not supported.
three.module.js:16000 THREE.WebGLRenderer: OES_texture_half_float extension not supported.
three.module.js:16000 THREE.WebGLRenderer: OES_texture_half_float_linear extension not supported.
three.module.js:16000 THREE.WebGLRenderer: OES_texture_float_linear extension not supported.
three.module.js:16000 THREE.WebGLRenderer: EXT_frag_depth extension not supported.
three.module.js:16000 THREE.WebGLRenderer: WEBGL_draw_buffers extension not supported.
three.module.js:16000 THREE.WebGLRenderer: EXT_shader_texture_lod extension not supported.
three.module.js:16000 THREE.WebGLRenderer: EXT_texture_filter_anisotropic extension not supported.
Attempting to present in AR...
```

#782 might be a related issue? Model is always black here tho.

### Browser Affected

- [x] Chrome
- [ ] Edge
- [ ] Firefox
- [ ] Helios
- [ ] IE
- [ ] Safari

### OS

- [x] Android
- [ ] iOS
- [ ] Linux
- [ ] MacOS
- [ ] Windows

**Comment by cdata** [comment]
Thanks for reporting, @takahirox ! It looks like this is the WebXR-based AR mode. Is that correct?

**Comment by takahirox** [comment]
Yes!

**Comment by cdata** [comment]
@takahirox thanks, this part of our code hasn't received much love, but we are hoping to shore it up soon. It's very interesting to see this, and I too wonder if it is a device-specific issue.

cc @elalish @bialpio

**Comment by takahirox** [comment]
I also tested with [WebXR Emulator Extension](https://github.com/MozillaReality/WebXR-emulator-extension) (AR support is WIP) on desktop windows Chrome and see a model black, too. Maybe non platform specific issue.

![image](https://user-images.githubusercontent.com/7637832/74365752-8a9fbb80-4d83-11ea-92fb-7f331ad5e3ae.png)

There seems be no lights in `ARRenderer` scene. I guess this would be the reason.

**Comment by takahirox** [comment]
The model is colored by adding `DirectionalLight` (color: 0xffffff, intensity: 5}.

![image](https://user-images.githubusercontent.com/7637832/74387366-34477280-4dad-11ea-9f78-b59c7658086e.png)

**Comment by cdata** [comment]
@takahirox thanks for the extra testing 🙌 

Yah, our AR renderer is considered unstable / experimental. Given that the issue is in WebXR-based AR, it would not surprise me to learn that recent changes to Three.js or our WebXR integration have broken it.

I'll try to take a look at the issue before our release.

**Comment by takahirox** [comment]
I investigated more and found the root issue.

A model can be colored even without `Light` node in a scene if `.environment` of top `scene` is set due to envitonment lighting. This is the reason why the model is colored out of AR even without `Light` node in non-AR `scene`.

But `.environment` property of only top `scene` works. In AR mode, the `scene` (I call `ModelScene`) is added under `ARRenderer`'s scene. And `ARRenderer` scene's `.environment` is `null`. So this is the reason why the model is black in AR-mode.

A solution may be copying `ModelScene`'s `.environment` to `ARRenderer` scene's. The screenshot is of the one I copied the `.environment`.

![image](https://user-images.githubusercontent.com/7637832/74488899-712c6b80-4e78-11ea-9017-05e3a4c4d28c.png)
